<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetGestureLearningDialog</class>
 <widget class="QDialog" name="TargetGestureLearningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Target Gesture Learning</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="stopButton">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="backButton">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QGridLayout" name="standardGestureLayout"/>
     </item>
     <item row="1" column="1">
      <layout class="QGridLayout" name="userGestureLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>INITING CAMERA...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
